obal/data/update_package.yml

```diff
@@ -47,3 +50,8 @@
       args:
         chdir: "{{ inventory_dir }}"
       when: new_source is defined
+
+    - name: 'Commit update'
+      command: "git commit -a -m 'Release {{ inventory_hostname }} {{ version }}'"
```

Comment:
In foreman-packaging I don't use ```-a``` when updating but explicitly add the package instead. Sometimes I have some tooling changes open and then it's nice not to accidentally commit those.